src/circadian_model.R

```diff
@@ -152,25 +141,49 @@ sleepState <- function(t, A, R1tot, Psi, params) {
       }
     }
 
-    # Set mu and ichi parameters based on whether student is awake or not
+    return (isAwake)
+
+
+}
+
+lightExposure <- function(t, isAwake, params) {
+    rise <- tanh(0.6 * ((t %%24) - 8.0))
+    fall <- tanh(0.6 * ((t %%24)- 17))
+    smoothLight <- (700.0/2.0) * (rise - fall) + 40.0
```

Comment:
Let's add parameters so that we can control the shape of the light exposure without changing the source code. Let's use 4 parameters called:

- `lightSlope` = 0.6
-  `lightRiseTime` = 8.0
-  `lightFallTime` = 17.0
-  `lightAmplitude` = 700.0
-  `lightBaseline` = 40.0

And the code will look something like:
```R
  rise <- tanh(lightSlope * ((t %% 24) - lightRiseTime))
  fall <- tanh(lightSlope * ((t %% 24)- lightRiseTime))
  smoothLight <- (lightAmplitude/2.0) * (rise - fall) + lightBaseline
```
don't forget to unpack the parameters and add them to `test_model.R`!